webapp-django/crashstats/crashstats/tests/test_models.py

```diff
@@ -735,17 +735,15 @@ def mocked_get(url, params, **options):
         ok_(r['hits'])
```

Comment:
Almost all the changes in this file is for changing the mocking now that the crash_data comes from an implementation instead of the middleware (json).

I bet you can ignore everything in this file.